_developer-files/connect/api/objects/form-properties/sources/saas/saasoptics-object.md

```diff
@@ -0,0 +1,70 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-saasoptics-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "SaaSOptics Source Form Property"
+api-type: "platform.saasoptics"
+display-name: "SaaSOptics"
+
+source-type: "saas"
+docs-name: "saasoptics" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "account_name"
+    type: "string"
+    required: true
+    description: |
+      The name of your unique SaaSOptics database schema, called an "account." This account is the one in which your subscription financial records are managed.
+    value: "<YOUR_ACCOUNT_NAME>"
+
+  - name: "server_subdomain"
+    type: "string"
+    required: true
+    description: "The subdomain of the SaaSOptics server where the account is running."
+    value: "<YOUR_SUBDOMAIN>"
+    
+  - name: "token"
+    type: "string"
+    required: true
+    description: |
+      The API token. This token allows Stitch to access your {{ form-property.diaplay-name }} account's API. Refer to the [{{ form-property.display-name }} documentation]({{ doc-link | append: "#obtain-token" }}) for instructions on retrieving this credential.
```

Comment:
The formatting looks great here ;)

A few suggestions:

1. `The API token.` > Perhaps `A SaaSOptics API token.`, to make it a bit more specific. In this case, I think it'll help orient the reader to the section they're looking at (SaaSOptics Form Property) due to the page being so long and the section header being so far above the table this is in.

2. This is nitpicky (sorry), but the API token doesn't allow Stitch to access the user's SaaSOptics' account API. It just lets Stitch query for their account's data using the SaaSOptics API. To be honest, I don't even think you need this sentence here - I'd take it out to make the description more concise.